client.go

```diff
@@ -367,16 +368,51 @@ func (c *Client) ImportFrame(frame *Frame, iterator RecordIterator, options ...I
 	return c.importManager.Run(frame, iterator, importOptions.withDefaults())
 }
 
-func (c *Client) ImportValueFrame(frame *Frame, field string, iterator RecordIterator, options ...ImportOption) error {
-	// c.importValues is the default importer for this function
-	ibf := func(indexName string, frameName string, slice uint64, rows []Record) error {
-		return c.importValues(indexName, frameName, slice, field, rows)
-	}
-	newOptions := []ImportOption{importBitsFunction(ibf)}
-	for _, opt := range options {
-		newOptions = append(newOptions, opt)
+type ValueIterator interface {
+	NextValue() (FieldValue, error)
+}
+
+// ImportValueFrame imports field values from the given iterator.
+func (c *Client) ImportValueFrame(frame *Frame, field string, valueIterator ValueIterator, batchSize uint) error {
```

Comment:
would there be any value in including the functional options here for batchSize? `options ...ImportOption`

Reply:
maybe - it will still break the PDK unfortunately, so I'll need to update that at the same time.